Sources/MediaViewController.swift

```diff
@@ -30,6 +31,14 @@ public class VLCMediaViewController: UICollectionViewController, UISearchResults
         dragAndDropManager.delegate = services.mediaDataSource
         return dragAndDropManager
     }()
+    
+    lazy var emptyView: VLCEmptyLibraryView = {
+        let name = String(describing: VLCEmptyLibraryView.self)
+        let nib = Bundle.main.loadNibNamed(name, owner: self, options: nil)
+        guard let emptyView = nib?.first as? VLCEmptyLibraryView else { fatalError("Can't find nib for \(name)") }
```

Comment:
I don't think we need to crash the app if we can't display an emptyview. An assertion and returning a UIView would've been just as fine